### ISSUE TITLE
"Error: Not connected" when navigating from models to the dashboard and then back to models

### ISSUE THREAD

**Issue by rumspeed** [beyondcode/invoker-community#75]
Connecting to a project and selecting any of the models works just fine. The "Error: Not connected" message comes up when navigating to the Dashboard, then back to the models. This is happening on a local project as well as a remote project. I just upgraded to v2.0.2. It was not happening when I was on v2.0.1.

------
**OS**: win32
**Invoker Version**: 2.0.2
**Laravel Version**: 8.20.1
**Local project**: unknown
**PHP Binary**: /usr/local/bin/php
**Route**: /project/index?class=App%5CModels%5CCustomer&time=1612385086972&project=remote-

**Error**:
```
Error: Not connected
```

**Comment by mpociot** [comment]
Do you have any widgets on your dashboard? If so, how many?
And do you also see a "not connected" error locally?

**Comment by rumspeed** [comment]
> Do you have any widgets on your dashboard? If so, how many?

Yes. I have seven widgets. All data displays just fine. I was able to add them when I was on v2.0.1. Now that I am on v2.0.2, I am not able to edit them or add new ones. Nothing saves.

> And do you also see a "not connected" error locally?

Well, I do have a local project however I am using Homestead on my Windows machine so I still connect to it using SSH. I am getting the same behavior with that project as well.

**Comment by mpociot** [comment]
Could you please provide me with a screenshot of your dev tools? You can access it from the "Developer" main menu

**Comment by mpociot** [comment]
@rumspeed nevermind - I was able to reproduce the issue by creating multiple widgets.
To give you some background on this issue: Invoker creates one SSH connection - this connection then executes multiple commands via SSH. The SSH daemon on your server has a maximum of allowed concurrent open "sessions" (executions from one connection). 
When loading multiple widgets, Invoker sends X commands via SSH, which causes the session limit to exceed - resulting in the error.

I fixed this by introducing a queue for these SSH commands so that only 3 concurrent commands will be executed.
I'll publish the updated version tomorrow/later today.
